src/main/groovy/nextflow/trace/GraphObserver.groovy

```diff
@@ -118,4 +118,9 @@ class GraphObserver implements TraceObserver {
     boolean enableMetrics() {
         return false
     }
+
+    @Override
+    void onFlowError(TaskHandler handler, TraceRecord trace) {
+
+    }
 }
```

Comment:
These can be removed because now it's a trait, right?

Reply:
arg, forgot to remove it